src/commands/quit.json

```diff
@@ -6,6 +6,11 @@
         "since": "1.0.0",
         "arity": -1,
         "function": "quitCommand",
+        "deprecated_since": "7.2.0",
+        "replaced_by": "just closing the connection",
```

Comment:
@guybe7 did we have any plans for replaced_by to always point to a specific command (rather than just plain text)?

Reply:
we don't rely on it, but i'd rather replaced_by would always be a command name

Reply:
Currently, it is a string of text usually with some commands embedded within it. I think we should follow that until we change it for all commands. Some examples:

```
georadius_ro.json:        "replaced_by": "`GEOSEARCH` with the `BYRADIUS` argument",
georadiusbymember.json:        "replaced_by": "`GEOSEARCH` and `GEOSEARCHSTORE` with the `BYRADIUS` and `FROMMEMBER` arguments",
georadiusbymember_ro.json:        "replaced_by": "`GEOSEARCH` with the `BYRADIUS` and `FROMMEMBER` arguments",
```

On the redis.io website, it is rendered inside a sentence so we need to take care that the grammar ends up correct:

> As of Redis version **{deprecated_since}**, this command is regarded as deprecated.
>
> It can be replaced by **{replaced_by}** when migrating or writing new code.